libs/client/ui/pages/src/lib/exercise-detail/exercise-detail.tsx

```diff
@@ -119,18 +114,7 @@ export const ExerciseDetail: React.FC<ExerciseDetailProps> = inject(
 
     return (
       <IonPage className="exercise-detail">
-        <IonHeader mode="ios">
-          <IonToolbar>
-            <IonButtons>
-              <IonBackButton
-                color="primary"
-                defaultHref={tabRoutes.start.route}
-                text="Zurück"
-              />
-            </IonButtons>
-            <IonTitle>{park?.name}</IonTitle>
-          </IonToolbar>
-        </IonHeader>
+        <BackBtn />
```

Comment:
I find da button nice. aber er bugt (vor allem uf ios weil döt jo sliding animationen dinna sin). ma kann dean leider ned einfach so ihne rendern. musch manuell verzögert displayen wia is zb bei da tabbar gmacht hob.

    useIonViewWillEnter(() => {
      ...
    }, []);

    useIonViewWillLeave(() => {
      ...
    }, [])

Reply:
hoi, also bei mir bugt es bei Android net? nur bei Ios, aber danke das es sesch - hobs in ios nämlich net getestet keht.